### PR TITLE
PDCT-451 Added summary endpoint to new analytics route.

### PR DESCRIPTION
**Pull request by katybaulch**
# Description

- Added new analytics router & updated AuthEndpoint and the AUTH_TABLE.
- Added the 'analytics/summary' endpoint returning a dummy response (with the schema below).

```
{
  n_documents: int,
  n_families: int,
  n_collections: int,
  n_events: int,
}
```

Another PR to come to flesh this out with the correct data.

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [x ] New feature
- [ ] Breaking change

## How Has This Been Tested?

Tested locally using HTTPie - tests to come in next PR.

## Reviewer Checklist

- [ ] The PR represents a single feature (small driveby fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: app/clients/db/models/app/authorisation.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ class AuthOperation(str, enum.Enum):
 }
 
 
-class AuthEntity(str, enum.Enum):
+class AuthEndpoint(str, enum.Enum):
```
Comment (diversemix) [inline app/clients/db/models/app/authorisation.py:24]: Nice choice of name :+1:

File: app/service/authorisation.py [hunk 2]
```diff
@@ -22,15 +22,15 @@ def http_method_to_operation(method: str) -> AuthOperation:
     raise AuthorisationError(f"Unknown HTTP method {method}")
 
 
-def path_to_entity(path: str) -> AuthEntity:
+def path_to_entity(path: str) -> AuthEndpoint:
```
Comment (diversemix) [inline app/service/authorisation.py:25]: Tiny niggle - but fix it in the next PR .... we can change the function name to something like `path_to_auth_endpoint()`

**Review by diversemix — APPROVED** [review]
Looks great! Assumed you've tested locally :smile:

File: app/service/authorisation.py (hunk 2 above)
Comment (katybaulch) [reply app/service/authorisation.py]: Pushed to this PR 👍

**Pull request merged**